lib/produce/itunes_connect.rb

```diff
@@ -74,11 +74,14 @@ def fetch_apple_id
 
     def initial_create
       open_new_app_popup
-      
+
       # Fill out the initial information
       wait_for_elements("input[ng-model='createAppDetails.newApp.name.value']").first.set @config[:app_name]
       wait_for_elements("input[ng-model='createAppDetails.versionString.value']").first.set @config[:version]
       wait_for_elements("input[ng-model='createAppDetails.newApp.vendorId.value']").first.set @config[:sku]
+      if @config[:company_name] != ''
```

Comment:
At first I did it without `!= ''` but didn't work (I don't know why), which is why I had to add `!= ''`.